app/views/verify/address/index.html.slim

```diff
@@ -3,13 +3,27 @@ h1.h2
 p
   = decorated_session.verification_method_choice
 
-.clearfix
-  .sm-col.sm-col-12.md-col-6
-    = link_to t('idv.buttons.activate_by_phone'), verify_phone_path,
-      class: 'btn btn-primary mb2 center inline-block'
+= form_tag
+  label.block.p2.border.rounded-md.border-light-blue.mb2 for="address_delivery_method_phone"
+    .radio.mb2
+      = radio_button_tag 'address_delivery_method', :phone
+      span.indicator
+      strong.blue.block = t('idv.messages.select_verification_form.phone_header')
+      span = t('idv.messages.select_verification_form.phone_message')
+  
+  // USPS verification should always be enabled in prod.
+  // This check is a convenience for lower envs.
```

Comment:
We can probably drop the comment in here

Reply:
I think I may be more of a fan of comments than other people. My thinking was that since I had to stop and ask about it, leave a comment for the next person so they will know.

Reply:
We typically try to keep comments out the idp. Since there's a lot of code and a lot of churn, it is difficult to keep comments up to date. If you think there's a risk that someone will change the code and break something later, it may be better to add a tests. If you think the code is unclear, it may be better to refactor it so it is more clear.

This comment is an example of one that will go out of date and be confusing when we change this config in the lower envs.